heartbeat/exportfs

```diff
@@ -417,6 +417,10 @@ testdir() {
 }
 exportfs_validate_all ()
 {
+	if ! echo "$OCF_RESKEY_fsid" | grep -q -E "^[A-Za-z0-9-]*$"; then
+		ocf_exit_reason "$OCF_RESKEY_fsid is not a valid fsid"
+		return $OCF_ERR_CONFIGURED
+	fi
```

Comment:
Hmm, I see now that there's a check for this below (ocf_is_decimal). Anything wrong with it?